### PR TITLE
WFCORE-1361 Update the command synopsis for jboss-cli

### PR DESCRIPTION
**Pull request by aloubyansky**
Thanks,
Alexey

### CODE REVIEW

File: cli/src/main/resources/help/connect.txt [hunk 1]
```diff
@@ -34,7 +34,8 @@ DESCRIPTION
   - an IPv4 address, e.g. 127.0.0.1
   - an IPv6 address, e.g. [::1]
 
-
+  Controller addresses can be aliased and stored in jboss-cli.xml config file.
+  Aliases can later be used as arguments for connect command.
```
Comment (bstansberry) [inline cli/src/main/resources/help/connect.txt]: "for the connect command"

File: cli/src/main/resources/help/help.txt [hunk 2]
```diff
@@ -14,10 +16,13 @@ Usage:
  --version       - prints the version info of the JBoss AS release, JVM and
                    system environment.
 
- --controller    - the default controller host and port to connect to when
-                   --connect option (described below) is specified or when the
-                   connect command is issued w/o the arguments. The default
-                   controller host is localhost and the port is 9990.
+ --controller    - the default controller (including protocol, host and port)
+                   to connect to when --connect option (described below) is
```
Comment (bstansberry) [inline cli/src/main/resources/help/help.txt]: when the --connect option

File: cli/src/main/resources/help/help.txt [hunk 3]
```diff
@@ -77,7 +82,7 @@ Usage:
                    to return. The value provided must be a positive integer.
                    Defaults to 5000 milliseconds when not provided.
 
-  --bind         - specifies to which address the CLI is going to be bound to.
+ --bind          - specifies to which address the CLI is going to be bound to.
```
Comment (bstansberry) [inline cli/src/main/resources/help/help.txt]: is going to be bound.

No final "to" is needed as the clause begins with "to which".

File: cli/src/main/resources/help/connect.txt (hunk 1 above)
Comment (bstansberry) [inline cli/src/main/resources/help/connect.txt]: Also "store in the jboss-cli.xml config file"

**Comment by aloubyansky** [comment]
Thanks for corrections!

**Pull request merged**